### ISSUE TITLE
chartjs labelling style

### ISSUE THREAD

**Issue by sanathoi93** [chartjs/chartjs-plugin-datalabels#131]
![label](https://user-images.githubusercontent.com/47590338/56454205-d65e2000-637f-11e9-9300-dde062fa20fc.png)

Is there a way to get this style of labelling in chartjs.

**Comment by JohnArcher** [comment]
I am not sure whether/how this is possible with chartjs-plugin-datalabels, but you could use this plugin: https://piechart-outlabels.netlify.com

**Comment by simonbrunel** [comment]
@sanathoi93 are you talking about the links between the arc sections and labels?

**Comment by SSRico** [comment]
I am looking for the same style of labels, @simonbrunel Is it possible to connect the labels with the chart with a line as the screenshot added by sanathoi?

**Comment by simonbrunel** [comment]
@SSRico It's not possible yet, I have a pending branch but it's far from ready.

**Comment by SSRico** [comment]
@simonbrunel I hope the future version includes this feature, good to know you have some work in progress, thanks for the update.

**Comment by sanathoi93** [comment]
> @sanathoi93 are you talking about the links between the arc sections and labels?

yes. it looks cool to have some links between labels and its associated arcs

**Comment by Luc9802** [comment]
Any news on this? Is this a planned feature?

**Comment by silllli** [comment]
+1

**Comment by Uysim** [comment]
+1

**Comment by FlorysiakSimon** [comment]
+1

**Comment by johnzzon** [comment]
+1
